src/components/VacancyJobInformation.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { Box, Stack, Typography, TextField } from '@mui/material';
+
+export type VacancyJobInformationProps = {
+  onNext: () => void;
+  formData: {
+    jobName: string;
+    department: string;
+    tasksAndResponsibilities: string;
+    requiredSkills: string;
+    workplaceAndWorkingHours: string;
+    languageRequirements: string;
+    additionalInformation: string;
+    [key: string]: string;
+  };
+  setFormData: React.Dispatch<
+    React.SetStateAction<{
+      jobName: string;
+      department: string;
+      tasksAndResponsibilities: string;
+      requiredSkills: string;
+      workplaceAndWorkingHours: string;
+      languageRequirements: string;
+      additionalInformation: string;
+    }>
+  >;
+};
+
+const VacancyJobInformation: React.FC<VacancyJobInformationProps> = ({ formData, setFormData }) => {
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const { name, value } = event.target;
+    setFormData((prevData) => ({
+      ...prevData,
+      [name]: value,
+    }));
+  };
+
+  return (
+    <Box>
+      {/* Basic information form */}
+      <Stack spacing={2}>
+        <TextField
+          label="Job Name"
+          name="jobName"
+          value={formData.jobName}
+          onChange={handleChange}
+          fullWidth
+          required
+        />
+        {/* TODO: This should be a dropdown and the available departments should be fetched from backend (Departments should be switched to ENUM in backend and db)*/}
+        <TextField
+          label="Department/Responsibility Area"
+          name="department"
+          value={formData.department}
+          onChange={handleChange}
+          fullWidth
+          required
+        />
```

Comment:
See comment in TODO: this should be selectable via dropdown, since the various departments are from the beginning. I am not sure, if departments are currently still saved as string or if it was already changed to an enum in backend.

Reply:
Created dropdown field in the frontend, but need clarification on backend handling. Using a placeholder for now.

Reply:
Let´s also talk about this in the next meeting